packages/plugins/blocks/blocks-markdown/src/blocks/DangerousMarkdown/DangerousMarkdown.js

```diff
@@ -34,7 +35,8 @@ class DangerousMarkdown extends React.Component {
       <div id={blockId} className={methods.makeCssClass(properties.style)}>
         <ReactMarkdown
           className="markdown-body"
-          plugins={[gfm]}
+          remarkPlugins={[gfm]}
+          rehypePlugins={[rehypeRaw]}
           allowDangerousHtml={true}
```

Comment:
The `allowDangerousHtml ` property has been removed, please remove it from the component